lineage/figures/figureCommon.py

```diff
@@ -247,3 +252,96 @@ def figureMaker1(ax, x, paramEst, accuracies, tr, pii, paramTrues, xlabel="Numbe
     ax[i].set_title("Initial Probability Matrix Estimation")
     ax[i].grid(linestyle="--")
     ax[i].tick_params(axis="both", which="major", grid_alpha=0.25)
+
+def figureMaker2(ax, x, paramEst, accuracies, tr, pii, paramTrues, xlabel="Number of Cells"):
```

Comment:
If this is specific to one figure, it should just be in that file.

Reply:
This is for the figures we want to make when we have G1 and G2 separated.
Should I move only this function back to the figure file or the parameters, too?